code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -81,6 +81,8 @@
 					new /obj/item/tarot_card_pack/mega(src)
 				if(11)
 					new /obj/item/tarot_generator(src) // ~1/250? Seems reasonable
+		if(25)
```

Comment:
this case is impossible with current code
`var/loot = rand(1, 23)`

Reply:
That is what I get for removing everything else to test whether it spawns or not

Reply:
Loot range no longer a concern due to Everfull replacing Inferno clusterbang